### ISSUE TITLE
not compiling because of DB_CSS to DB_CCS change

### ISSUE THREAD

**Issue by mcshane** [thegenemyers/DEXTRACTOR#26]
`DB_CSS` was changed to `DB_CCS` in 917f6f1a8986c0e338a468155d2951bc3208e289, but was not updated in `dex2DB.c`. Does not compile currently.

```diff --git a/dex2DB.c b/dex2DB.c
index 22f965d..76ab8c5 100644
--- a/dex2DB.c
+++ b/dex2DB.c
@@ -647,7 +647,7 @@ int main(int argc, char *argv[])
                 offset += clen;

                 if (pwell == s->well)
-                  { prec[pcnt].flags |= DB_CSS;
+                  { prec[pcnt].flags |= DB_CCS;
                     pcnt += 1;
                     if (pcnt >= pmax)
                       { pmax = ((int) (pcnt*1.2)) + 100;
@@ -843,7 +843,7 @@ int main(int argc, char *argv[])
                 offset += clen;

                 if (pwell == rec->well)
-                  { prec[pcnt].flags |= DB_CSS;
+                  { prec[pcnt].flags |= DB_CCS;
                     pcnt += 1;
                     if (pcnt >= pmax)
                       { pmax = ((int) (pcnt*1.2)) + 100;
```

**Comment by thegenemyers** [comment]
So sorry.  Fixed and checked in.

Shane this is the old version of the scrubber that caused breaks in 
micro-satellitic regions
due to excessive error rates.  It is also designed for long subreads and 
not HiFi reads which
makes quite a difference.  I do have improved versions for both cases 
but never took either
to the finish line and made them available.   I could get one or the 
other of these two to the
point of usability by a third party with a week's work.  I offered this 
before but never got
sufficient specifications of what you wanted to proceed.  I'd be willing 
to do this if it is
important to you (i.e. you are still trying to fix the missed adaptamer 
subreads that were
discussed the last time I was at Hinxton.  I know you like to just get 
on and often go it alone,
but I offer to try connecting on this again if your willing.

     Best,  Gene

**Comment by mcshane** [comment]
Thanks @thegenemyers. Note DEXTRACTOR here also needs updating with the same fix. Not just in DASCRUBBER.

Sorry, I kind of dropped off with the palindrome adapters thing. Yes, would be good to restart that a work again. Will send an email.

**Comment by blaiseli** [comment]
I have a related compilation failure with the version cloned from github yesterday:
```
dex2DB.c: In function ‘main’:
dex2DB.c:650:41: error: ‘DB_CSS’ undeclared (first use in this function); did you mean ‘DB_CCS’?
  650 |                   { prec[pcnt].flags |= DB_CSS;
      |                                         ^~~~~~
      |                                         DB_CCS
dex2DB.c:650:41: note: each undeclared identifier is reported only once for each function it appears in
make: *** [Makefile:36: dex2DB] Error 1
```

As a side note, on my debian-based linux, I had to patch the Makefile so that hdf5 is correctly found.
I added the following two variables:
```
PATH_HDF5_INCLUDE = /usr/include/hdf5/serial
PATH_HDF5_LIB = /usr/lib/x86_64-linux-gnu/hdf5/serial
```
And then replaced every occurrence of `$(PATH_HDF5)/include` and `$(PATH_HDF5)/lib` with `$(PATH_HDF5_INCLUDE)` and `$(PATH_HDF5_LIB)` respectively.

**Comment by thegenemyers** [comment]
Yes, it is DB_CCS.  Not sure why such an error is there.  -- Gene
